src/lib/tree/tree.md

```diff
@@ -0,0 +1,50 @@
+The `mat-tree` provides a Material Design styled tree that can be used to display hierarchy
+data.
+
+This tree builds on the foundation of the CDK tree and uses a similar interface for its
+data source input and template, except that its element and attribute selectors will be prefixed 
+with `mat-` instead of `cdk-`.
```

Comment:
Do you plan on adding examples in a follow-up PR?

Reply:
Yes. I can add placeholders